Wox.Core/Updater.cs

```diff
@@ -10,22 +10,25 @@
 using Squirrel;
 using Newtonsoft.Json;
 using Wox.Core.Resource;
+using Wox.Plugin.SharedCommands;
 using Wox.Infrastructure;
 using Wox.Infrastructure.Http;
 using Wox.Infrastructure.Logger;
+using System.IO;
 
 namespace Wox.Core
 {
     public class Updater
     {
         public string GitHubRepository { get; }
+        private bool IsPortableMode => Directory.Exists(Constant.PortableDataPath);
```

Comment:
I think we should move this to Constant (since it is already doing it while checking for data directory).
In the future I think we should get rid of constant....

Reply:
sure